### ISSUE TITLE
How do I get Tracker ID?

### ISSUE THREAD

**Issue by Lydecker** [distriqt/ANE-Firebase#152]
I would like to use Firebase Analytics for Android however I can't find trackerID which I need for the CustomResource.ANE.

If I download my google-services.json file from my Firebase Console, this is all I get for Analytics:

![image](https://user-images.githubusercontent.com/12632602/41046724-d30b3000-69a2-11e8-9134-f05f0f9ead3f.png)

no trackerID!

I have an android analytics dashboard working on my account..... 

Any ideas?

Thanks

**Comment by Lydecker** [comment]
OH - I'm assuming this is because I never used Google Analytics? Am I correct in thinking you only get a tracker ID if you migrate Google Analytics over to Firebase Analytics? But if you are creating a new app in Firebase which has never used GA, then you don't need the tracker_id ?

Thanks

**Comment by marchbold** [comment]
Yeah the standard configuration using Firebase doesn't contain a GA tracker id. You are correct that this is only for migrating older accounts.
